homeassistant/components/climate/opentherm_gw.py

```diff
@@ -0,0 +1,201 @@
+"""OpenTherm Gateway Climate component for Home Assistant."""
+
+import logging
+import voluptuous as vol
+
+from homeassistant.components.climate import (ClimateDevice, PLATFORM_SCHEMA,
+                                              STATE_IDLE, STATE_HEAT,
+                                              STATE_COOL,
+                                              SUPPORT_TARGET_TEMPERATURE)
+from homeassistant.components.climate.modbus import CONF_PRECISION
```

Comment:
Don't import things from other platforms.

Reply:
addressed in c5a9266871b32a981ad19ab38a0bad650ab644e4